lib/text/ttml_text_parser.js

```diff
@@ -239,12 +245,13 @@ shaka.text.TtmlTextParser = class {
    * @param {!Array.<!shaka.text.CueRegion>} cueRegions
    * @param {boolean} whitespaceTrim
    * @param {boolean} isNested
+   * @param {?Object} cellResolution
```

Comment:
"Object" is too general a type.  Please use a record type like you did in Cue:
`{?{columns: number, rows: number}}`, so that the compiler can catch a caller using the wrong fields.